### ISSUE TITLE
Forge Installer

### ISSUE THREAD

**Issue by jp9ronaldo** [Lebroncraft/crewmod#1]
I dont have an application that can install the Forge-Installer, What would be a good one from the microsoft store? Sorry, Im new at mods, Thanks!!

**Comment by Abysss** [comment]
Java???

**Comment by ThePizzaGuy277** [comment]
Yeah, I can't download the forge installer client AT ALL, it says I need to run the version 1.7.10 at least once, but i have 1.8.3 :(

**Comment by jp9ronaldo** [comment]
That's what happened to me. So I redownloaded everything

**Comment by ThePizzaGuy277** [comment]
Everything, including Minecraft itself?

**Comment by jp9ronaldo** [comment]
No. Just the mod stuff

**Comment by ThePizzaGuy277** [comment]
Ohhhhh yeah Im stupid :P

**Comment by ThePizzaGuy277** [comment]
That didn't work either :(

**Comment by jp9ronaldo** [comment]
Anyone know how to fix the "You need to run the version 1.7.10 at least once problem?

**Comment by ThePizzaGuy277** [comment]
Okay I got it :D so you need to actually run Minecraft in 1.7.10. You can do this by going to the Minecraft launcher, and in the bottom left should be something that says "Edit Profile". Now, once you go to that screen, in the "Update Notes" section, you should find a section that says "Version Selection". Now, from there, where it says "Allow use of old 'Alpha' Minecraft versions (from 2010)", click the little button. Then, where it says "Use Version" go to the list and select "release 1.7.10". It should work now :D

**Comment by jp9ronaldo** [comment]
That runs normal minecraft. I can't run a single player mod server. Does anyone know how to do it?

**Comment by eevee27** [comment]
I have downloaded forge 1.7.10 an i have it up running but when i'm putting the mod pack into the mods folder it docent register and when i log on to minecraft it only says that i have 3 mods: Minecraft Forge, Forge Mod Loader and Minecraft coder pack. I'm kinda new to this stuff and i'm on a macbook air. Anyone that can help me? :(

**Comment by ThePizzaGuy277** [comment]
Yeah same thing with me eevee

**Comment by jp9ronaldo** [comment]
Ok. I think I figured it out. (Mine works fine). Forget ALL of the directions! Download forge. Remove the mods folder from the client. Replace it with the mods file in the ".minecraft" folder. (You can find that by searching your computer). Good luck!!

**Comment by ThePizzaGuy277** [comment]
That didn't work either. Can the devs come on and help us or something?

**Comment by Mitsab** [comment]
im not a dev but just follow the instructions.txt
